backend/README.md

```diff
@@ -0,0 +1,112 @@
+# Measure Repository Service
+
+A prototype implementation of a [FHIR Measure Repository Service](https://build.fhir.org/ig/HL7/cqf-measures/measure-repository-service.html)
+
+## Installation
+
+### Prerequisites
+
+- [Node.js >=16.0.0](https://nodejs.org/en/)
+- [MongoDB >= 6.0](https://www.mongodb.com)
+
+### Local Installation
+
+Clone the source code:
+
+```bash
+git clone https://github.com/projecttacoma/measure-repository-service.git
+```
+
+Install dependencies:
+
+```bash
+npm install
```

Comment:
Your call on whether this makes sense to add or not.. since we have the `git clone` section in here I was reading the instructions from the perspective of git cloning and then needing to cd into the backend directory to get things installed there

Reply:
Doesn't hurt to add!